cobigen-cli/src/main/java/com/devonfw/cobigen/cli/commands/CobiGenCommand.java

```diff
@@ -46,30 +50,58 @@ public void run() {
 	}
 
 	/**
-	 * This class implement getVersion() and this method return the version of plug-in
+	 * This class implement getVersion() and this method return the version of
+	 * plug-in
 	 */
 	static class PropertiesVersionProvider implements IVersionProvider {
 		@Override
 		public String[] getVersion() throws Exception {
-			String versionProvider[] = new String[50];
+			List<String> versionProvider = new ArrayList<String>();
 			MavenXpp3Reader reader = new MavenXpp3Reader();
-			Model model;
-			if ((new File("pom.xml")).exists())
-				model = reader.read(new FileReader("pom.xml"));
-			else
-				model = reader.read(
-						new InputStreamReader(CobiGenCommand.class.getResourceAsStream("/src/main/resources/pom.xml")));
+			Model model = null;
+			File locationCLI = new File(CobiGenUtils.class.getProtectionDomain().getCodeSource().getLocation().toURI());
```

Comment:
Instead of `CobiGenUtils` can you use `CobiGenCommand`? It is the class that we currently are :)